backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Metadaten.java

```diff
@@ -54,7 +54,7 @@ public Optional<String> getFrameSimpleMetadatum(final T simpleMetadatum, final L
    * @param date - the specific date
    * @param newValue - the new value to update/create
    */
-  public void updateSimpleMetadatum(
+  public void updateFrameSimpleMetadatum(
```

Comment:
This doesn't look like a good renaming, since this method is also use for metadata within einzelelement right? I would just revert the name.

Reply:
True

Reply:
Same for `getFrameSimpleMetadatum` -> `getSimpleMetadatum`